### PR TITLE
Added flag to only broadcast when playing

### PR DESCRIPTION
**Pull request by taahamahdi**
I was planning to name the flag `--no-pause` to match `--no-idle`, but the `-p` substring of `--no-pause` was interfering with the port number flag. The new flag is `--unpaused`.

### CODE REVIEW

File: src/main.cpp [hunk 1]
```diff
@@ -214,9 +227,10 @@ int main(int argc, char** args)
     auto port = getPort(vecArgs);
 
     AppState state = {};
-    state.BroadcastIdle     = !isFlagSet(vecArgs, "--no-idle");
-    state.UseMultipleApps   = isFlagSet(vecArgs, "--use-multiple-apps");
-    
+    state.BroadcastIdle     = shouldBroadcastIdle(vecArgs);
+    state.BroadcastPaused   = shouldBroadcastPaused(vecArgs);
+    state.UseMultipleApps   = shouldUseMultiAppModel(vecArgs);
```
Comment (SSStormy) [inline src/main.cpp:232]: The calls to `isFlagSet` were fine as they were.
(I do realize that shouldBroadcastIdle and shouldBroadcastPaused weren't used before)

File: src/main.cpp [hunk 2]
```diff
@@ -204,6 +216,7 @@ int main(int argc, char** args)
         LINE("  -P=PASSWORD             the password to be sent after connection to the MPD server has been established in hopes of acquiring higher permissions. (default is empty, therfore no password sent.)");
         LINE("  --fork                  forks the process into the background.");
         LINE("  --no-idle               Disables broadcasting of the idle state.");
+        LINE("  --unpaused              Disables broadcasting of the paused state.");
```
Comment (SSStormy) [inline src/main.cpp:219]: `--unpaused` is ambiguous. How about `--dont-broadcast-the-paused-state`?

File: src/main.cpp (hunk 1 above)
Comment (SSStormy) [inline src/main.cpp:232]: The calls to isFlagSet were fine as they were. No point in calling to should*
(I do realize that shouldBroadcastIdle and shouldBroadcastPaused weren't used before)

File: src/main.cpp [hunk 3]
```diff
@@ -160,6 +167,11 @@ bool shouldBroadcastIdle(const std::vector<std::string>& args)
     return !isFlagSet(args, "--no-idle");
 }
 
+bool shouldBroadcastPaused(const std::vector<std::string>& args)
+{
+    return !isFlagSet(args, "--unpaused");
```
Comment (SSStormy) [inline src/main.cpp:172]: Ditto

**Comment by taahamahdi** [comment]
Sorry for the wait! I've made the corrections you suggested

**Comment by taahamahdi** [comment]
Hey -- I was just thinking about how we decided to implement the change, and `--dont-broadcast-the-paused-state` won't actually work in this case either. Sorry about not realizing this earlier! I get a `std::invalid_argument` when passing ``--dont-broadcast-the-paused-state` :[

The reason it won't work is because of the `-p` substring: won't this get mistaken for `-p`'s port option too, just like when `--no-pause` didn't work?

**Comment by taahamahdi** [comment]
Would `--unpaused-broadcast-only` work for you?

**Comment by SSStormy** [comment]
ced628d3eaf3f18c5eff286b0955c605616348ee should fix it.

**Pull request merged**